debug_toolbar/toolbar.py

```diff
@@ -106,7 +107,10 @@ def should_render_panels(self):
             # incompatible with the toolbar until
             # https://github.com/jazzband/django-debug-toolbar/issues/1430
             # is resolved.
-            render_panels = self.request.META.get("wsgi.multiprocess", True)
+            if isinstance(self.request, ASGIRequest):
+                render_panels = False
```

Comment:
This conditional doesn't align with the comment above it. If the comment is to be trusted, then we can change the default value to False and avoid this check on `ASGIRequest`.